pt-BR/mail/chrome/messenger/am-server-advanced.dtd

```diff
@@ -22,7 +22,7 @@
 <!ENTITY overrideNamespaces.label "Permitir que o servidor sobrescreva estes namespaces">
 <!ENTITY overrideNamespaces.accesskey "e">
 <!ENTITY pop3Desc.label "Ao receber mensagens POP deste servidor, salvá-las na:" >
-<!ENTITY folderStorage.label "Where to store mail">
+<!ENTITY folderStorage.label "Onde armazenar emails">
```

Comment:
E-mail is a PTBR word. It was incorporated and is currently in the dictionary. After talking to Sherman, we decided to change all occurrences of "email" to "e-mail".